org.freedesktop.GstDebugViewer.json

```diff
@@ -1,42 +1,53 @@
 {
-    "app-id" : "org.freedesktop.GstDebugViewer",
-    "branch" : "stable",
-    "runtime" : "org.gnome.Platform",
-    "runtime-version" : "43",
-    "sdk" : "org.gnome.Sdk",
-    "command" : "gst-debug-viewer",
-    "rename-icon" : "gst-debug-viewer",
-    "copy-icon" : true,
-    "finish-args" : [
-        "--socket=x11",
+    "app-id": "org.freedesktop.GstDebugViewer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "command": "gst-debug-viewer",
+    "rename-icon": "gst-debug-viewer",
+    "copy-icon": true,
+    "finish-args": [
+        "--device=dri",
         "--share=ipc",
-        "--socket=wayland",
-        "--filesystem=xdg-run/dconf",
-        "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
-        "--talk-name=org.freedesktop.Notifications",
-        "--filesystem=host",
```

Comment:
Could we keep at least one filesystem permission here? This app can't possibly work without any of those...

Reply:
The open file dialogue gives the app access to the selected file using the portal.
For the command line there is the `--file-forwarding` option.

Reply:
It is gonna work through portal but I bet most users launch it on the command line and requiring passing `--file-forwarding` is suboptimal fmopv - I think giving home access by default is reasonable.

Reply:
Okay, I'll add `--filesystem=home:ro`